laalaa/settings/local.py.example

```diff
@@ -0,0 +1,4 @@
+from .common import *
```

Comment:
This fails for me with `No module named common` :(

Reply:
It seems we can remove the import and the `TEMPLATE_DEBUG` too:

`?: (1_8.W001) The standalone TEMPLATE_* settings were deprecated in Django 1.8 and the TEMPLATES dictionary takes precedence. You must put the values of the following settings into your default TEMPLATES dict: TEMPLATE_DEBUG.`.

Reply:
Agreed. Good spot.